### DISCUSSION TITLE
What happens if a mount fails

### DISCUSSION THREAD

**Discussion by justcop** [alexbelgium/hassio-addons#1236] [Q&A]
I'm wondering what the behaviour would be in the following situation.
Lets say I set up Sonarr or any other addon to download to mounted local external drive eg /mnt/Storage
If the mount fails because for example the drive has been disconnected, what will happen?

Will Sonarr still write to /mnt/Storage, but this will now be a local file instead of on the external disk. If so, how do I prevent this, so that the mount point isn't blocked, if the disk isn't correctly mounted?

**Comment by alexbelgium** [comment]
Hi, this line should provide the script to continue (or at least log it an issue): https://github.com/alexbelgium/hassio-addons/blob/f63978eb4ace5d2e8d3bbc45b22c524bc8a385e6/.templates/00-smb_mounts.sh#L17
